projects/wm-core/src/filters/status-filter/status-filter.component.ts

```diff
@@ -62,6 +64,8 @@ export class StatusFilterComponent {
   }
 
   resetFilters(): void {
+    this._store.dispatch(goToHome());
```

Comment:
cosi va bene ma ti invito a pensare che potremmo fare un dispatch go to home che chiama a sua volta questi due dispatch come effect, questo sposta la logica nello state managment e focalizza il dispatch in una sola azione